templates/contribs.html

```diff
@@ -12,7 +12,9 @@ <h1>People whom have contributed to MirahezeBot</h1>
                 <li><a href="https://meta.miraheze.org/wiki/User:Zppix">Zppix</a> - Founder and former Head of Operations</li><br/>
                 <li><a href="https://meta.miraheze.org/wiki/User:Reception123">Reception123</a> - Sysadmin</li><br/>
                 <li><a href="https://meta.miraheze.org/wiki/User:Paladox">Paladox</a> - Assistant Developer and Advisor</li><br/>
-                <li><a href="https://github.com/hasandiwan">Hasan Diwan</a> - Advisor and Reviewer</li>
+                <li><a href="https://github.com/hasandiwan">Hasan Diwan</a> - Advisor and Reviewer</li><br/>
+                <li><a href="https://meta.miraheze.org/wiki/User:Sario528">Sario</a> - Assistant Developer and Test Analyst</li><br/>
+                <li><a herd="https:/:meta.mirahezr.org/wiki/User:Dmehus">Doug</a> - Community Relations Specialist and PA to the Technical Lead</li><br/>
```

Comment:
Should be `https://` not `https:/:`

Reply:
@Universal-Omega Good spot. Also `a href` not `a herd`.

Reply:
I'd have had better luck slamming my head on my keyboard. Thanks!

Reply:
Oh right. Thanks!